softwerkskammer/frontend/javascript/activityDateModel.js

```diff
@@ -9,8 +8,13 @@ var activityDateModel;
   activityDateModel = function (initialDate, initialTime) {
 
     var toUtc = function (dateString, timeString) {
+      // returns javascript Date
+      function stringToInt(each) { return parseInt(each, 10); }
```

Comment:
Da kann auch NaN rauskommen... Wäre es nicht besser, das prophylaktisch in irgendwas unschädliches umzuwandeln? Z. B. 
```suggestion
      function stringToInt(each) { 
          const result = parseInt(each, 10); 
          return isNaN(result) ? 0 : result;
      }
```